fre/app/generate_time_averages/frenctoolsTimeAverager.py

```diff
@@ -17,7 +17,17 @@ class inheriting from abstract base class timeAverager
     '''
 
     def generate_timavg(self, infile=None, outfile=None):
-        ''' use fre-nctool's CLI timavg.csh with subprocess call '''
+        """
+        use fre-nctool's CLI timavg.csh with subprocess call
+
+        :param self: 
+        :param infile: path to history file, or list of paths
+        :type infile: str, list
+        :param outfile: path to where output file should be stored
+        :type outfile: str
+        :param self: time scale for climatology. Accepted variables vary based on pkg ('all','seas','month') 
+        :type self: str, defaults to 'all'. Inherted from generate_time_averages.py
+        """
```

Comment:
I see some ValueErros raised in this function.  My recommendation is to include that here:
```
:raises ValueError:

    - instance variable self.avg_type not supported
    - No infile specified
    - Cannot find timavg.csh (likely the user is not in an environment with frenctools installed)
    - timavgcsh command is not properly executed
```

I also see we return an exit code:
```
:return: 1 if timavg.csh command is not properly executed, and 0 if function has a clean exit
:rtype: int
```